docs/source/configuration.rst

```diff
@@ -16,9 +16,9 @@ Configuring Calico
 ==================
 
 This page describes how to configure Calico. We first describe the
-configuration of the core Calico component - Felix -
+configuration of the core Calico component -- Felix --
 because this is needed, and configured similarly, regardless of the
-surrounding environment (OpenStack, Docker, or whatever). Then,
+surrounding environment (OpenStack, Docker, or whatever.) Then,
```

Comment:
I misread this; for some reason I thought it was a sentence entirely in parentheses. Oops; fix coming.